README.md

```diff
@@ -47,36 +44,27 @@ subscribers to those tags via the external services.
 * GOV.UK Notify API key and other details (see
   [`email_service.yml`](config/email_service.yml) for required fields)
 
-### Initial setup
-
-* Check that the configuration in `config/database.yml` is correct
-* Run `bundle exec rake db:setup` to load the database
-
 ### Running the application
 
-```bash
-$ ./startup.sh
-```
-
-* email-alert-api runs on port 3088
-* sidekiq-monitoring for email-alert-api uses 3089
+Using any email address that ends with `@notifications.service.gov.uk`
+will not create a subscriber or a subscription, however will return a `201 Created` response.
 
 ### Running the test suite
 
 * Run `RAILS_ENV=test bundle exec rake db:setup` to load the database
```

Comment:
Might want to lose the bullet point here, now this is a lone bullet.

Reply:
Oops - meant to delete that. Updated.